charts/hush-sensor/templates/hushsecrets.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
```

Comment:
The name of the file should be `hushsecret` as it creates a `Secret`.

Reply:
Changed name to deploymentsecret.yaml.